docs/mkdocs.yml

```diff
@@ -22,69 +22,79 @@ plugins:
-  - Dremio: https://docs.dremio.com/data-formats/apache-iceberg/
-  - Starrocks: https://docs.starrocks.io/en-us/latest/data_source/catalog/iceberg_catalog
-  - Amoro: amoro.md
-  - Amazon Athena: https://docs.aws.amazon.com/athena/latest/ug/querying-iceberg.html
-  - Amazon EMR: https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-iceberg-use-cluster.html
-  - Amazon Data Firehose: https://docs.aws.amazon.com/firehose/latest/dev/apache-iceberg-destination.html
-  - Amazon Redshift: https://docs.aws.amazon.com/redshift/latest/dg/querying-iceberg.html
-  - Google BigQuery: https://cloud.google.com/bigquery/docs/iceberg-tables
-  - Snowflake: https://docs.snowflake.com/en/user-guide/tables-iceberg
-  - Impala: https://impala.apache.org/docs/build/html/topics/impala_iceberg.html
-  - Databend: https://docs.databend.com/guides/access-data-lake/iceberg
-  - Doris: https://doris.apache.org/docs/dev/lakehouse/catalogs/iceberg-catalog
-  - Druid: https://druid.apache.org/docs/latest/development/extensions-contrib/iceberg/
-  - DuckDB: https://duckdb.org/docs/preview/core_extensions/iceberg/overview
-  - Kafka Connect: kafka-connect.md
+  - Concepts:
+    - Tables:
+      - branching.md
+      - configuration.md
+      - evolution.md
+      - maintenance.md
+      - metrics-reporting.md
+      - partitioning.md
+      - performance.md
+      - reliability.md
+      - schemas.md
+    - Views:
+      - view-configuration.md
+  - Libraries:
+    - Java:
+      - Quickstart: java-api-quickstart.md
+      - API: api.md
+      - Javadoc: ../../javadoc/latest/
+    - Python: https://py.iceberg.apache.org/
+    - Rust: https://rust.iceberg.apache.org/
+    - Go: https://pkg.go.dev/github.com/apache/iceberg-go/
   - Integrations:
-    - aws.md
-    - dell.md
+    # Iceberg-maintained integrations come first
+    - Spark:
+      - spark-getting-started.md
+      - spark-configuration.md
+      - spark-ddl.md
+      - spark-procedures.md
+      - spark-queries.md
+      - spark-structured-streaming.md
+      - spark-writes.md
+    - Flink:
+      - flink.md
+      - flink-connector.md
+      - flink-ddl.md
+      - flink-queries.md
+      - flink-writes.md
+      - flink-actions.md
+      - flink-configuration.md
+    - Kafka Connect: kafka-connect.md
+    - Third-party:
+    # Now all the 3rd party integrations
+      - Amazon Athena: https://docs.aws.amazon.com/athena/latest/ug/querying-iceberg.html
+      - Amazon Data Firehose: https://docs.aws.amazon.com/firehose/latest/dev/apache-iceberg-destination.html
+      - Amazon EMR: https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-iceberg-use-cluster.html
+      - Amazon Redshift: https://docs.aws.amazon.com/redshift/latest/dg/querying-iceberg.html
+      - Apache Doris: https://doris.apache.org/docs/dev/lakehouse/catalogs/iceberg-catalog
+      - Apache Druid: https://druid.apache.org/docs/latest/development/extensions-contrib/iceberg/
+      - Apache Hive: hive.md
```

Comment:
This is not all third-party. HiveCatalog is maintainted by Iceberg community.

Reply:
Fixed ac97a9e711cd52efb304d34dc2f0e32779473065